webapp/portlet/src/main/webapp/vue-apps/common/js/ActivityService.js

```diff
@@ -1,5 +1,25 @@
-export function getActivities(spaceId, limit, expand) {
-  return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/v1/social/activities?spaceId=${spaceId || ''}&limit=${limit}&expand=${expand || ''}`, {
+export function getActivities(streamType, limit, expand) {
+  let params = {};
+
+  if (eXo.env.portal.spaceId) {
+    params.spaceId = eXo.env.portal.spaceId;
+  }
```

Comment:
Avoid changing JS method API by using contextual parameters, like using `ConversationState` in Service layer in Backend. So please use `spaceId` method parameter instead of contextual parameter `window.eXo.env.portal.spaceId`